.github/workflows/promptflow-executor-e2e-test.yml

```diff
@@ -161,5 +161,4 @@ jobs:
         osVersion: ubuntu-latest
         pythonVersion: 3.9
         coverageThreshold: 70
-        token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
as the CI passed, I'm curious on why this workflow can pass without specify `token`?

Reply:
we don't use the token at all.